src/components/CatalogContext.js

```diff
@@ -15,9 +15,11 @@ class CatalogContext extends Component {
         theme,
         title,
         pages: pages.filter((p) => !p.hideFromMenu),
+        pagePaths: new Set(pages.map((p) => p.path)),
```

Comment:
Actually, it's just to have a fast way to valodate internal links. Config validation is an important topic, I agree. But I'll save it for another PR :grin: